studies/forms.py

```diff
@@ -539,3 +544,22 @@ class ExternalForm(ModelForm):
     class Meta:
         model = Study
         fields = ()
+
+
+class JSPsychForm(ModelForm):
+    experiment = forms.CharField(
+        label="jsPsych Experiment Code",
+        widget=AceOverlayWidget(
+            mode="javascript",
+            wordwrap=True,
+            theme="textmate",
+            width="100%",
+            height="100%",
+            showprintmargin=False,
+        ),
+        help_text="Please enter jsPsych experiment code.",
```

Comment:
This isn't important for now but I want to write it down so that we don't forget. We might want to make sure that users know that this box is just for the jsPsych JavaScript and not for HTML (since jsPsych users are probably used to putting their experiment in an HTML file with the actual code in script tags, and might try to do the same thing here). We'll also need to let them know what plugins are available. We could add a bit more help text here and/or add a link to our jsPsych docs.

Reply:
How about something like: "Please enter jsPsych experiment code. See the \<a href="jspsych-docs-link">CHS jsPsych documentation\</a> for more information about how to add your code and what plugins are available."

Reply:
I'll make a note in the looki-docs jsPsych issue about adding this link once the docs page exists!